common/common.go

```diff
@@ -273,6 +273,8 @@ const (
 	// EnvServerSideDiff defines the env var used to enable ServerSide Diff feature.
 	// If defined, value must be "true" or "false".
 	EnvServerSideDiff = "ARGOCD_APPLICATION_CONTROLLER_SERVER_SIDE_DIFF"
+	// EnvArgoCDgRPCMaxSizeMB is the environment variable to look for a max gRPC message size
+	EnvArgoCDgRPCMaxSizeMB = "ARGOCD_GRPC_MAX_SIZE_MB"
```

Comment:
Just a nit, and I know that it was like this before, but I think we do not need to include "ArgoCD" in the environment variable name. Also, I find the casing for gRPC odd in the context of a variable name. Can we make it either `EnvGrpcMaxSize` or `EnvGRPCMaxSize` (I prefer the former)?

Reply:
The problem, that this variable already exist, and it will be breaking change

Reply:
I'm not talking about the name of the environment variable itself (`ARGOCD_GRPC_MAX_SIZE_MB`), but the variable name used in the code (`EnvArgoCDgRPCMaxSizeMB`). I don't see renaming the Go variable to be a breaking change.

Reply:
sorry, my initial comment might have been misleading, because I wrote "environment variable name"

Reply:
So, to clarify:

`EnvArgoCDgRPCMaxSizeMB` should rather be `EnvGrpcMaxSizeMB` or `EnvGRPCMaxSizeMB`, with a preference to the former.

Reply:
Done @jannfis